docs/elements.md

```diff
@@ -2317,10 +2317,12 @@ Displays the contents of question directions.
 
 #### Details
 
-Contents are only shown during question input portion. When a student
-either makes a submission or receives the correct answer, the information
-between these tags is hidden. If content exists outside of a question panel,
-then it will be displayed alongside the answer.
+Information between this tag represents the question text and is 
+always shown when the question is displayed to students. When a 
```

Comment:
I don't think this is precise enough - we consider the submission and answer panels to also be "displaying the question to students".

An alternative formulation of this text that we can iterate on:

> The contents of this element will only be displayed in the question panel, which generally displays the question prompt and any inputs that will be used to submit answers. The contents will not be shown when the question is rendered to display the correct answer or a particular submission.

Reply:
Okay, I've taken another crack at it:

> Generally, any text placed in `question.html` will appear in all of `pl-question-panel`, `pl-submission-panel`, and `pl-answer-panel`. To prevent content from being displayed in `pl-submission-panel` and `pl-answer-panel`, surround the content with the `<pl-question-panel>` tags.

Feel free to say you like your attempt better, I can iterate on that instead!

Reply:
This sounds like a confusion between the elements (e.g., pl-submission-panel) with the panels (the submission panel).

Reply:
Yes you're right, thanks! Made those edits in the comment.

Reply:
Your version works for me! I'll leave a few more comments after you make the change.

I'd recommend ensuring that the docs for the other `*-panel` tags is consistent with this. In a perfect (and hopefully future) world, we'd have documentation that explains panels, and then we can just have the docs for all these elements be something like this:

> Ensures that content is only displayed in the question panel. See (link to panel documentation) for more details).

That can wait for the inevitable docs revamp 🙂